src/lib/transakService.ts

```diff
@@ -297,3 +299,47 @@ async function updateFeeSavings(
     console.error(error);
   }
 }
+
+async function getTransakApiKey(address: string) {
+  const tierInfo = await getWalletTierInfo(address);
+  const isTopTier = tierInfo.tier <= 2;
+  const apiKey = TRANSAK_API_KEYS[isTopTier ? 1 : 0];
+  return apiKey;
+}
+
+export async function createTransakWidgetUrl(
+  widgetParams: Record<string, unknown>,
+  referrerDomain: string
+) {
+  const apiKey = await getTransakApiKey(widgetParams.walletAddress as string);
+  const accessToken = await getValidAccessToken(apiKey);
+  const response = await fetch(
+    "https://api-gateway-stg.transak.com/api/v2/auth/session",
```

Comment:
Update to prod API once PR is approved.

Reply:
Can we also move this hardcoded URL into a constant like you did with BASE_URL above?